deployment/network-operator/templates/operator.yaml

```diff
@@ -78,6 +78,8 @@ spec:
               value: "network-operator"
             - name: ENABLE_WEBHOOKS
               value: "{{ .Values.operator.admissionController.enabled }}"
+            - name: USE_DTK
```

Comment:
should we have an option to disable it via values ?

Reply:
Added